pkg/cli/root.go

```diff
@@ -435,11 +439,50 @@ func validateConfiguration(config *config.Config) error {
 	return nil
 }
 
+func validateOpenIDConfig(config *config.Config) error {
+	if config.HTTP.Auth != nil && config.HTTP.Auth.OpenID != nil {
+		for provider, providerConfig := range config.HTTP.Auth.OpenID.Providers {
+			//nolint: gocritic
+			if api.IsOpenIDSupported(provider) {
+				if providerConfig.ClientID == "" || providerConfig.Issuer == "" ||
+					len(providerConfig.Scopes) == 0 {
+					log.Error().Err(errors.ErrBadConfig).
+						Msg("OpenID provider config requires clientid, issuer and scopes parameters")
+
+					return errors.ErrBadConfig
+				}
+			} else if api.IsOauth2Supported(provider) {
+				if providerConfig.ClientID == "" || len(providerConfig.Scopes) == 0 {
+					log.Error().Err(errors.ErrBadConfig).
+						Msg("OAuth2 provider config requires clientid and scopes parameters")
+
+					return errors.ErrBadConfig
+				}
+			} else {
+				log.Error().Err(errors.ErrBadConfig).
+					Msg("unsupported openid/oauth2 provider")
+
+				return errors.ErrBadConfig
+			}
+		}
+
+		if config.Extensions == nil || config.Extensions.APIKey == nil ||
```

Comment:
We can remove the check. API keys and OpenID are 2 independent features, which could be used separately.